yb-voyager/src/utils/commonVariables.go

```diff
@@ -100,7 +100,8 @@ type DBObject struct {
 }
 
 // TODO: support MinimumVersionsFixedIn in xml
-type Issue struct {
+type AnalyzeSchemaIssue struct {
```

Comment:
I believe you have just changed the struct name, not the JSON field names, but have we tested YugabyteD to confirm if this change does not break anything?

Reply:
its the struct name only, not the fields. So json will be same.
We send payload(json) to ybd